lib/surface/compiler.ex

```diff
@@ -615,9 +615,7 @@ defmodule Surface.Compiler do
 
   defp validate_tag_children([%AST.Template{name: name} | _]) do
     {:error,
-     "templates are only allowed as children elements of components, but found template for #{
-       name
-     }"}
+     "templates are only allowed as children elements of components, but found template for #{name}"}
```

Comment:
formatter seems to have changed a few lines. I can remove if we want

Reply:
ah i see why, i'm on elixir 1.12 and its formatting differently there. i'll send another PR to set `.tool-versions`